gensrc/script/functions.py

```diff
@@ -249,7 +249,9 @@
     [30080, 'repeat', 'VARCHAR', ['VARCHAR', 'INT'], 'StringFunctions::repeat'],
 
     [30090, 'lpad', 'VARCHAR', ['VARCHAR', 'INT', 'VARCHAR'], 'StringFunctions::lpad', 'StringFunctions::pad_prepare', 'StringFunctions::pad_close'],
+    [30091, 'lpad', 'VARCHAR', ['VARCHAR', 'INT'], 'StringFunctions::lpad', 'StringFunctions::pad_prepare', 'StringFunctions::pad_close'],
```

Comment:
this line can removed now

Reply:
Is it necessary to keep this line so that `show full builtin functions;` can show functions with different numbers of parameters.
mysql> show full builtin functions like 'lpad';
+---------------------------+-------------+---------------+-------------------+-----------------------------------------------------------------+
| Signature                 | Return Type | Function Type | Intermediate Type | Properties                                                      |
+---------------------------+-------------+---------------+-------------------+-----------------------------------------------------------------+
| lpad(VARCHAR,INT)         | VARCHAR     | Scalar        | NULL              | {"fid":"30091","symbol":"","file":"","type":"BUILTIN","md5":""} |
| lpad(VARCHAR,INT,VARCHAR) | VARCHAR     | Scalar        | NULL              | {"fid":"30090","symbol":"","file":"","type":"BUILTIN","md5":""} |
+---------------------------+-------------+---------------+-------------------+-----------------------------------------------------------------+